pyaqueduct/client/client.py

```diff
@@ -412,14 +436,19 @@ def get_extensions(self) -> List[ExtensionData]:
                 error.errors if error.errors else "Unknown error occurred in the remote operation."
             ) from error
 
-        extensions = list(map(ExtensionData.from_dict, extensions["extensions"]))  # pylint: disable=unsubscriptable-object
-        logging.info("Fetched %s extensions", len(extensions))
-        return extensions
+        extensions_list = list(
+            map(
+                ExtensionData.from_dict,
+                extensions_response["extensions"],  # pylint: disable=unsubscriptable-object
+            )
+        )
+        logging.info("Fetched %s extensions", len(extensions_list))
+        return extensions_list
```

Comment:
This change is probably irrelevant to the task, but ok

Reply:
It was not directly related, but someone forgot to autoformat the code before pushing and because this file is changed in my PR, autoformatter has fixed it too.